spec/dsl_spec.rb

```diff
@@ -46,7 +45,19 @@ def target_class_name
 
   sig { returns(String) }
   def target_class_file
-    target_class_name.underscore
+    underscore(target_class_name)
+  end
+
+  sig { params(class_name: String).returns(String) }
```

Comment:
This seems to be replication of the previous declaration of this method in `lib/tapioca/generator.rb`. Would it be better to make this a helper module somewhere we can inject instead?

Reply:
Indeed it is, but since this is a test base class, I wasn't too worried about refactoring to a helper class/module. The real usage is in `generator.rb` only.